enhancements/api-review/add-ImagePolicy-to-config.openshift.io-v1alpha1.md

```diff
@@ -0,0 +1,294 @@
+  - https://issues.redhat.com/browse/OCPNODE-1628
+---
+
+# Add CRD ImagePolicy to config.openshift.io/v1alpha1
+
+## Summary
+
+This enhancement propose an ImagePolicy CRD to provide a way to set sigstore image signature verification configuration on Openshift.
+
+## Motivation
+
+### User Stories
+
+As an OpenShift user, I want to verify the container images signed using sigstore
+tools, so that I can utilize the increased security of my software supply chain.
+
+### Goals
+
+- The user can create an ImagePolicy instance specifying the images/repositories to be verified and their policy. The MCO will write the configuration for signature verification. Once this is done, CRI-O can verify the images/repositories.
+
+- MCO container runtime config controller watches ImagePolicy instance in different namespaces and merges the instances for each namespace into a single [containers-policy.json](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md) in a predefined `/path/to/policies/\<NAMESPACE\>.json`.
+
+- CRI-O can verify Cosign signature signed images using configuration from ImagePolicy by using the namespace from the sandbox config on the `PullImage` RPC.
+
+### Non-Goals
+
+## Proposal
+
+### Workflow Description
+
+**cluster administrator** is a human user responsible for adding signature verification configurations in
+cluster scope.
+1. The cluster administrator creates ImagePolicy yaml file with `namespace: openshift-config`.
+2. The cluster administrator creates ImagePolicy instance using `oc create -f imagepolicy.yaml`.
+3. The cluster administrator can get the merged cluster scope policies from `status` of ImagePolicy instances. 
+4. The cluster administrator can delete the signature verification configuration by deleting its ImagePolicy instances. 
+
+**application administrator** is a human user responsible for
+adding signature verification configurations for specific namespace.
+
+1. The application administrator creates ImagePolicy yaml file for namespace that is not `namespace: openshift-config`
+2. The application administrator creates ImagePolicy instance using `oc create -f imagepolicy.yaml`.
+3. The application administrator can get the merged cluster policy and policy for certain namespace from `status` of ImagePolicy instances. 
+4. The application administrator can delete the signature verification configuration by deleting its ImagePolicy instances.
+5. The application administrator cannot override cluster scoped policies because they will treated with a higher priority.
+
+### API Extensions
+
+#### Type definitions
+
+```go
+type ImagePolicySpec struct {
+	// image defines the list of images/repositories assigned to a policy.
+	Images []ImageScope `json:"images"`
+	// policy defines the verification policy
+	Policy Policy `json:"policy"`
+}
+
+// ImageScope is the item of the images list.
+// https://go.dev/play/p/FfL_3bpqSX5
```

Comment:
What are the Go Playground's retention policies?  It might be safer to inline your generation script in the commit message, instead of, or in addition to, including this link?